coins/mec.json

```diff
@@ -112,8 +110,8 @@
                 }
             }
         },
-        "hash_genesis_block": "f9e3f18eaa2404b23c57ed9961d7e777e95a1f2c88dfd373e5ddadc8ed343cc1",
+        "hash_genesis_block": "7520788e2d99eec7cf6cf7315577e1268e177fff94cb0a7caf6a458ceeea9ac2",
```

Comment:
why is hash_genesis_block changing?

Reply:
See https://github.com/LIMXTEC/Megacoin/blob/d18a29cb9acc14b05713e69a34685e2ac97f01a5/src/chainparams.cpp#L172

Reply:
message_magic="MegaCoin Signed Message:" see https://github.com/LIMXTEC/Megacoin/blob/66428979986d547ca506f742741328f11ee03972/src/validation.cpp#L98